eclipse-collections-api/src/main/java/org/eclipse/collections/api/multimap/bag/MutableBagIterableMultimap.java

```diff
@@ -35,6 +35,24 @@ default MutableBagIterableMultimap<K, V> withKeyValue(K key, V value)
         return this;
     }
 
+    /**
+     * Puts the key / values combination into the MutableBagIterableMultimap and returns
+     * the MutableBagIterableMultimap (this).
+     * @since 11.1
+     */
```

Comment:
I think you only need javadoc at the top. You should check me on this, but I don't think that copy/pasting the Javadoc on a sub-interface changes the rendered html in any way.

Reply:
Addressed.